Update developer-setup.md to provide no-docker setup (#1351)
mattermost/mattermost-developer-documentation: site/content/contribute/developer-setup.md

   sudo apt install build-essential
   ```

> 1. Install [Docker](https://www.docker.com/). If you dont want to use Docker, you can follow the steps in the sub-bullets for manual database setup

Reviewer: ```suggestion
1. Install [Docker](https://www.docker.com/). If you don't want to use Docker, you can follow the steps in the sub-bullets below for manual database setup.
```

>     1. Zet `MM_NO_DOCKER` to `true` in `server/config.mk` file

Reviewer: ```suggestion
    1. Set `MM_NO_DOCKER` to `true` in the `server/config.mk` file.
```

    2. Install [PostgreSQL](https://www.postgresql.org/download/)
    3. Run `psql postgres`. Then create `mmuser` by running `CREATE ROLE mmuser WITH LOGIN PASSWORD 'mostest';`

>     4. Modify the role to give rights to create ` database by running `ALTER ROLE mmuser CREATEDB;`

Reviewer: ```suggestion
    4. Modify the role to give rights to create a database by running `ALTER ROLE mmuser CREATEDB;`
```

>     5. Check the role by running `\du`

Reviewer: ```suggestion
    5. Confirm the role rights by running `\du`
```

    6. Before creating the database, exit by running `\q`

>     7. Login again via mmuser by running `psql postgres -U mmuser`

Reviewer: ```suggestion
    7. Login again via `mmuser` by running `psql postgres -U mmuser`
```
